vpr/src/pack/greedy_candidate_selector.h

```diff
@@ -513,6 +537,28 @@ class GreedyCandidateSelector {
     ///        dimension is the number of external outputs of the molecule.
     std::vector<std::vector<t_pack_molecule *>> unrelated_clustering_data_;
 
+    /// @brief Placement information of the atoms in the netlist known before
+    ///        packing.
+    const FlatPlacementInfo& flat_placement_info_;
+
+    /**
+     * @brief Bins containing molecules in the same tile. Used for pre-computing
+     *        flat placement information for clustering.
+     */
+    struct FlatTileMoleculeList {
+        // A list of molecules in the undefined sub-tile at this tile.
```

Comment:
Explain what the undefined sub-tile is, and say it is not used yet. (I'd also put it second in the variable list).